streams/src/test/java/org/apache/kafka/streams/integration/IQv2StoreIntegrationTest.java

```diff
@@ -630,29 +609,11 @@ public void shouldHandlePingQuery() {
                 .withQuery(query)
                 .withPartitions(mkSet(0, 1))
                 .withPositionBound(PositionBound.at(INPUT_POSITION));
-
         final StateQueryResult<KeyValueIterator<Integer, V>> result =
             IntegrationTestUtils.iqv2WaitForResult(kafkaStreams, request);
 
         if (result.getGlobalResult() != null) {
-            final QueryResult<KeyValueIterator<Integer, V>> queryResult = result.getGlobalResult();
-            final boolean failure = queryResult.isFailure();
-            if (failure) {
-                throw new AssertionError(queryResult.toString());
-            }
-            assertThat(queryResult.isSuccess(), is(true));
-
-            assertThrows(IllegalArgumentException.class, queryResult::getFailureReason);
-            assertThrows(IllegalArgumentException.class,
-                queryResult::getFailureMessage);
-
-            final KeyValueIterator<Integer, V> iterator = queryResult.getResult();
-            final Set<Integer> actualValue = new HashSet<>();
-            while (iterator.hasNext()) {
-                actualValue.add(valueExtactor.apply(iterator.next().value));
-            }
-            assertThat(actualValue, is(expectedValue));
-            assertThat(queryResult.getExecutionInfo(), is(empty()));
+            fail("global tables aren't implemented");
```

Comment:
This was covered in a prior PR. It will be fixed in https://issues.apache.org/jira/browse/KAFKA-13523